virt/virt-4-11-release-notes.adoc

```diff
@@ -194,6 +194,34 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-11-known-issues"]
 == Known issues
 
+* Updating to {VirtProductName} {VirtVersion} from version 4.10 is blocked until you install the standalone Kubernetes NMState Operator. This occurs even if your cluster configuration does not use any link:https://nmstate.io/[nmstate] resources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2126537[*BZ#2126537*])
+** As a workaround:
+. Verify that there are no node network configuration policies defined on the cluster:
++
+[source,terminal]
+----
+$ oc get nncp
+----
+. If the list is empty, annotate the `HyperConverged` custom resource (CR). The following command overwrites any existing JSON patches:
++
```

Comment:
What if the list is not empty, and some policies still exist?
a. I think that we should be explicit and say that the polices must be deleted in order to un-block the upgrade.
b. I think we should suggest a method of re-using the policies after upgrade, if the user needs them to remain available (for example - storing each policy in a yaml file, and applying this file on the upgraded cluster).

Reply:
@yossisegev IIUC, the workaround described in this release note only applies if there are are no nmstate resources in use. If there are existing node network policies, then the user is expected to proceed with installing the NMState Operator and complete the upgrade procedure. The Kubernetes NMState Operator preserves the existing nmstate configuration after upgrade.

@phoracek Is that correct?

Reply:
@sjhala-ccs I see your point.
In this case - my point b (suggest a method of re-using policies after upgrade)is indeed redundant.
I still think, however. that we should explicit, and suggest the user what to do if policies do exist:
Either -
a. Delete them
or
b. Install KNO and preserve the policies (along with all other nmstate capabilities).

Reply:
Okay, I will incorporate your suggestion and update the PR. Thanks.

Reply:
@yossisegev Please re-review: http://file.rdu.redhat.com/sjhala/BZ2126537-rn-known-issue/virt/virt-4-11-release-notes.html#virt-4-11-known-issues

Reply:
@sjhala-ccs sorry I'm too late here, but I would ask for the note about policy removal to be removed. If users have policies, they most likely need them. Removing them would mean that the network they use will one day disappear. If they use NNCP and want to upgrade, they should install the standalone operator.